### ISSUE TITLE
CI: Setup automatic testing of the conda packages

### ISSUE THREAD

**Issue by certik** [lcompilers/lpython#2292]
We have to setup automatic testing at a CI (in a new github project) to test the conda packages on all platforms, running our integration tests and ensuring they run correctly. That would reveal that it was missing lpython.py.

This should be done in a new separate github project.

**Comment by adam-urbanczyk** [comment]
I don't think that you need a separate project - you can simply extend tests in the conda package spec (see above).

**Comment by certik** [comment]
Thanks for the PR https://github.com/conda-forge/lpython-feedstock/pull/16, that's much better than nothing. 

What we also need is to test all our integration tests, not just one, which is what this issue is about.

**Comment by adam-urbanczyk** [comment]
I guess then it is a matter of adding `python run_tests.py --skip-run-with-dbg` to the `test` section. The point of the PR was to demonstrate that conda build is perfectly capable of running tests - no need to crate new projects

**Comment by certik** [comment]
We started a repository here: https://github.com/lcompilers/lpython-conda-testing, see https://github.com/lcompilers/lpython-conda-testing/issues/1.
